server/src/validation/auth.ts

```diff
@@ -0,0 +1,28 @@
+import { Request, Response, NextFunction } from 'express';
+
+import errorGenerator from 'utils/error/error-generator';
+import errorHandler from 'utils/error/error-handler';
+
+interface IReqBody {
+  id: string;
+  password: string;
+}
+
+export const signInValidateion = (req: Request, res: Response, next: NextFunction): void => {
```

Comment:
```suggestion
export const signInValidation = (req: Request, res: Response, next: NextFunction): void => {
```
오타요 !! 그런데 validation 미들웨어 있으니까 참 좋네요 ㅎㅎ

Reply:
오타...